[2023-08-18T13:55:19.038+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: serial_parallel_data_pipeline.P2 manual__2023-08-18T13:54:48.097929+00:00 [queued]>
[2023-08-18T13:55:19.080+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: serial_parallel_data_pipeline.P2 manual__2023-08-18T13:54:48.097929+00:00 [queued]>
[2023-08-18T13:55:19.085+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-18T13:55:19.148+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): P2> on 2023-08-18 13:54:48.097929+00:00
[2023-08-18T13:55:19.172+0000] {standard_task_runner.py:57} INFO - Started process 972 to run task
[2023-08-18T13:55:19.203+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'serial_parallel_data_pipeline', 'P2', 'manual__2023-08-18T13:54:48.097929+00:00', '--job-id', '1729', '--raw', '--subdir', 'DAGS_FOLDER/data_pipleline_serial_and_parallel.py', '--cfg-path', '/tmp/tmp9bhjbmpw']
[2023-08-18T13:55:19.206+0000] {standard_task_runner.py:85} INFO - Job 1729: Subtask P2
[2023-08-18T13:55:19.262+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.9/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-08-18T13:55:19.474+0000] {task_command.py:410} INFO - Running <TaskInstance: serial_parallel_data_pipeline.P2 manual__2023-08-18T13:54:48.097929+00:00 [running]> on host d4028c5ba2ac
[2023-08-18T13:55:20.058+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='serial_parallel_data_pipeline' AIRFLOW_CTX_TASK_ID='P2' AIRFLOW_CTX_EXECUTION_DATE='2023-08-18T13:54:48.097929+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-18T13:54:48.097929+00:00'
[2023-08-18T13:55:20.060+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-08-18T13:55:20.060+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 10']
[2023-08-18T13:55:20.150+0000] {subprocess.py:86} INFO - Output:
[2023-08-18T13:55:30.181+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-08-18T13:55:30.207+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=serial_parallel_data_pipeline, task_id=P2, execution_date=20230818T135448, start_date=20230818T135519, end_date=20230818T135530
[2023-08-18T13:55:30.265+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-18T13:55:30.293+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
